doc/api/style_api.rst

```diff
@@ -2,6 +2,10 @@
 ``matplotlib.style``
 ********************
 
+.. seealso::
+
+   Examples on using style sheets with :func:`matplotlib.style.use` can be found at :doc:`/gallery/style_sheets/style_sheets_reference`.
```

Comment:
"Examples of..."

Is there some reason this needs a "seealso"?  

Is there any reason to not be even a bit more verbose here?  I don't find the style sheet reference example particularly helpful either.  I'd suggest pointing to the style sheet section of the examples rather than the master reference.  And if you really want to make it nice, I'd include a bit more detail.  i.e.
```
Matplotlib has the ability to make style sheets that control the default values of variables in the matplotlibrc (see plt.rcParam) that control how plot elements look.  Style sheets allow different parts of a script to have different styles using the `styles.use('mystyle')` call. Users can also define their own custom styles and store them in their `.matplotlib` directory.  
```

Or something like that.  I'm a little surprised we don't already have anything like that.  

If you don't feel like tackling that here, thats fine, this is an improvement as-is (except for the typo), and we could open a new issue about being more verbose here..

Reply:
I understand. I think I could fix this typo and work on giving more detail in a separate PR.